### PR TITLE
new Color.md file

### PR DESCRIPTION
**Pull request by flodega**
### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by flodega** [comment]
@SSwiniarski The issue is gone now.  Thanks for the help.  Awaiting final review now

**Review by SSwiniarski — APPROVED** [review]

File: content/swiftui/concepts/views/terms/color/color.md
Comment (KTom101) [inline content/swiftui/concepts/views/terms/color/color.md:35]: There is a lot of extra spacing in this code example, can we change this to remove the extraneous spacing? In Codecademy Docs entries, we typically use 2-spaces for indentation in code examples.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@flodega I've added a second review of this.

File: content/swiftui/concepts/views/terms/color/color.md
Comment (flodega) [reply content/swiftui/concepts/views/terms/color/color.md:31]: @KTom101 What do you mean by this ?  Do you want me to remove that line of code entirely ?

**Comment by flodega** [comment]
@KTom101 I made some changes.  Please let me know if it is okay now.

File: content/swiftui/concepts/views/terms/color/color.md
Comment (KTom101) [reply content/swiftui/concepts/views/terms/color/color.md:32]: @flodega You made the right correction. Line breaks are added before and after subheadings. 👍

**Review by KTom101 — APPROVED** [review]
@flodega This looks good. I made a small change directly in commit [865a804](https://github.com/Codecademy/docs/pull/1675/commits/865a8044f92e537f6c6b7179a6745b75819c3cac). The only thing that needs to be done now is fixing the [failed checks](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md#i-submitted-a-pr-but-tests-are-failing-how-do-i-fix-them:~:text=I%20submitted%20a%20PR%20but%20tests%20are%20failing%2C%20how%20do%20I%20fix%20them%3F). Will you be able to do this yourself or would you like me to do it?

**Comment by flodega** [comment]
@KTom101 Thanks, and it would be helpful if you could help with the checks as I am currently not even sure why they are appearing.  I would also very much like to learn how you fix those said failed checks.  Thanks

**Comment by KTom101** [comment]
@flodega I already have npm installed so I ran `npm run format:verify` and `npm run format:all`  in my terminal on my local repo and pushed the changes up. You can see what was changed in commit [42b0c38](https://github.com/Codecademy/docs/pull/1675/commits/42b0c381173650993e088337a3a734f5ee1351d6), which were a couple of minor extra spacings that was causing the check errors.

**Comment by KTom101** [comment]
@flodega Link is live here: https://www.codecademy.com/resources/docs/swiftui/views/color You're currently listed as an anonymous contributor:

Screen Shot 2023-03-02 at 5 01 15 PM

If you would like to be listed follow these steps to add your GitHub username to your Codecademy Profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

YourGitHubName

2.) Then log out and log back into Codecademy using your GitHub login:

ReLogIn

< **Note:** Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to properly work. If you use the same email for both GitHub and Codecademy already, then you're set.

**Pull request merged**
